ocaml-lsp-server/src/code_actions/open_related.mli

```diff
@@ -0,0 +1,7 @@
+open Import
```

Comment:
The current convention for code action-containing files is to have prefix `action_`. 

I find it verbose, but it makes looking the files up easier (cmd + p and `action_`). 

We can remove prefixes in other files or add it here.

Reply:
The directory already includes the suffix `_action`. that's probably enough. I won't touch the convention here though.